README.md

```diff
@@ -8,7 +8,7 @@ Supported Grafana versions: 5.3.3+
 
 # Screenshots
 
-![Screenshot](https://github.com/CorpGlory/grafana-multibar-graph-panel/blob/master/src/screenshots/screenshot-1.png)
+![Screenshot](https://github.com/CorpGlory/grafana-multibar-graph-panel/raw/master/src/screenshots/screenshot-1.png)
```

Comment:
@gnought can you please explain this fix,

Reply:
There is an installed panel widget in grafana home dashboard. This fix is to fix the image path shown in http://grafana:3000/plugins/corpglory-multibar-graph-panel/